### ISSUE TITLE
Multi Class Multi Instance Image Classification Problem

### ISSUE THREAD

**Issue by suraj-deshmukh** [fchollet/keras#1528]
Hi, I am working on multi class multi instance classification problem and dataset has 5 classes and 2000 natural images. The dataset is provided on this http://lamda.nju.edu.cn/data_MIMLimage.ashx link. The dataset images has at least 1 instance and at most 3 instances associated with it.
I am using convolutional neural network with two fully connected neural network at the end for classification. I have used sigmoid activation function and binary cross entropy as loss function. I am getting 75% testing accuracy but when I use model.predict function on testing data i get output as a vector of real numbers between 0 and 1. 
for eg Y (actual class) = [1,0,0,1,0] and Y(predicted) =[0.7,0.012,0.02,0.5,0.165] So my question is how do i modify my code to get desired output in terms of zeros's and ones's.
My code 
[ConvNet.txt](https://github.com/fchollet/keras/files/100443/ConvNet.txt)
Ans also i don't know on which basis my model is giving me 75%(test accuracy) accuracy on this type of dataset?
